.gitignore

```
@@ -29,3 +29,6 @@ plugin/target/
 .rust-toolchain
 /rust-toolchain.toml
 /kubectl-openebs
+
+# artifacts
+artifacts/
```

only at the root right?

```suggestion
/artifacts/
```

---

Yes